test/box/misc.test.lua

```diff
@@ -325,21 +325,21 @@ s:drop()
 -- no errors.
 --
 
--- Without argument it is equivalent to new_tuple_format({})
-tuple_format = box.internal.new_tuple_format()
+-- Without argument it is equivalent to box.tuple.format.new({})
+tuple_format = box.tuple.format.new()
```

Comment:
It seems, it shouldn't be in the second patch.

Reply:
Fixed, thanks!